nisystemlink/clients/file/utilities.py

```diff
@@ -0,0 +1,35 @@
+"""Utilities for FileClient."""
+
+from nisystemlink.clients.file import FileClient
+from nisystemlink.clients.file.models import UpdateMetadataRequest
+
+
+def get_file_id_from_uri(uri: str) -> str:
+    """Get the File ID from a file URI.
+
+    Example: URI "/nifile/v1/service-groups/Default/files/5b874c4adedd0f1c78a22a96" returns "5b874c4adedd0f1c78a22a96"
+
+    Args:
+        uri: URI of a file on SystemLink
+
+    Returns:
+        ID of the File extracted from the `uri`.
+    """
+    # Split the uri by '/' and get the last part
+    parts = uri.split("/")
+    return parts[-1]
+
+
+def rename_file(client: FileClient, file_id: str, name: str) -> None:
```

Comment:
Our clients have been mostly 1 to 1 with the HTTP API, but I think I'd rather this be a method on the FileClient itself instead of a separate function.